Extend contribution notes, add submodule information, change syntax and fix typos (#12)
sclorg/welcome: README.md

# Welcome to the SCL.org Space

> This is an overview of the sclorg organization, aka all one can find here.

Reviewer: Please do not use 'aka'. Use the full names.

Author: I rewrote some sentences to make them sound more professional :smile:

# Welcome to the SCL.org Space

Why SCL.org? Since all the repositories, under this organization are somehow connected to the Software Collections concept. Software Collections give you the power to build, install, and use multiple versions of software on the same system, without affecting system-wide installed packages.
This is an overview of the sclorg organization, aka all one can find here.

> And since we have had set of new web development tools available as RPMs, tested and available in the directory on [softwarecollections.org](https://www.softwarecollections.org/en/), it is a good idea to re-use those tested packages and make container images from them, that are working fine in OpenShift.

Reviewer: Why did you delete the reference? It is not valid?

Author: It is still there on line 7. Git diff moved it to the wrong line (under removed Repositories Overview).

> # Welcome to the SCL.org Space

Reviewer 2: ```suggestion
# Welcome to the Software Collections Organization
```

> This document offers a comprehensive introduction to the sclorg organization, highlighting the repositories under our umbrella and providing examples of their practical applications, along with links to relevant external resources.

Reviewer 2: ```suggestion
This document offers a comprehensive introduction to the Software Collections Organization (SCLOrg), highlighting the repositories under our umbrella and providing examples of their practical applications, along with links to relevant external resources.
```

> The primary focus of SCL.org revolves around the Software Collections concept. Software Collections offer a streamlined approach to building, installing, and utilizing multiple software versions on a single system, without disrupting pre-existing system-wide package installation.

Reviewer 2: ```suggestion
The primary focus of SCLOrg revolves around the Software Collections concept. Software Collections offer a streamlined approach to building, installing, and utilizing multiple software versions on a single system, without disrupting pre-existing system-wide package installation.
```

> In light of recent developments, we have introduced a series of RPMs web development tools. These tools have been thoroughly tested and are accessible via the softwarecollections.org directory. To maximize the utility of these verified packages, we encourage the creation of container images compatible with OpenShift.

Reviewer 2: I have no information about publishing our work on softwarecollections.org. Maybe this was in the history and works no longer for us?
@pkubatrh do you know something more about this, please?

Reviewer 2: Thanks for removing the "softwarecollections.org". However, I think that removing it causes the sentence to loose its meaning. I suggest removing whole sentence.

Reviewer 2: But wait a little please, I will rather ask Peto about this.

Reviewer 2: So the result is we really can remove it.
I saw one more occurrence of softwarecollections.org in README.md. Would you mind deleting all the occurrences please?

Author: If it is confirmed, I can remove it. Thank you for the review. I will commit the changes.

## Repository Overview

> - [Container Contribution Guidelines](contribution.md): Learn about what principles to follow when contributing to the sclorg organization, particularly in relation to container repositories.

Reviewer 2: ```suggestion
- [Container Contribution Guidelines](contribution.md): Learn about what principles to follow when contributing to the SCLOrg, particularly in relation to container repositories.
```
AFAIK, in general we use capital letters like this for SCLOrg. This applies also for other occurrences of 'sclorg' in this text.

- [centpkg for SCLo](https://github.com/sclorg/centpkg-sclo): A compact wrapper that assists [SCLo SIG](http://wiki.centos.org/SpecialInterestGroup/SCLo) maintainers in building SCL packages in [CBS](http://cbs.centos.org/).
- [centos-release package](https://github.com/sclorg/centos-release-scl): A minimal RPM package with a YUM repository, offering upstream Software Collections built in [CBS](http://cbs.centos.org/) by the community (excluding those in RHSCL).
- [SCLo CI Tests](https://github.com/sclorg/sclo-ci-tests): Tests executed in [CentOS CI](http://ci.centos.org) to verify the successful (re)building of RPM packages in [CBS](http://cbs.centos.org/).
- [RPM List Builder](https://github.com/sclorg/rpm-list-builder): A tool designed to facilitate the rebuilding of SRPMs in [CBS](http://cbs.centos.org/) that were previously available in different repositories (e.g., RHSCL and others).
- [RHSCL Container CI](https://github.com/sclorg/rhscl-container-ci): Scripts for testing containers on the RHEL platform.

> # Where to Learn More About Containers

Reviewer 2: In my opinion is redundant to state such a generic information in SCLOrg's readme.

There are a lot of articles about Docker and containers generally, these are just a few that seem to provide a basic introduction:

- https://developer.fedoraproject.org/tools/docker/about.html
- https://fedoramagazine.org/quick-containers-with-fedora-dockerfiles/
- https://www.liquidweb.com/kb/how-to-list-and-attach-to-docker-containers/
- https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/overview_of_containers_in_red_hat_systems/introduction_to_linux_containers

> ## How to write a Dockerfile

Reviewer 2: The same for writing dockerfile - this is imo not information, that should be stated in SCLOrg's readme, as is not really related to SCLOrg stuff.
